alephmarcreader/abstractalephmarcreader.py

```diff
@@ -353,4 +362,117 @@ def get_recipient_organisation(self):
 
         return recipient_organisation
 
-    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
\ No newline at end of file
+    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
+
+    def get_supplement_remarks(self):
+        """
+        Returns comments on supplementing material following the letter. (See also Description.supplement.)
+        :return: [str]
+        """
+        remarks = []
+
+        for field in self.__get_field('525'):
+            footnote_text = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'a'))
+
+            if footnote_text:
+                remarks.append(footnote_text)
+
+        return remarks
+
+    get_supplement_remarks.__annotations__ = {'return': [str]}
+
+    def get_document_state(self):
+        """
+        Returns a description of the document state. (i.e. "original", "copy", "draft", etc.)
+        :return: [str]
+        """
+        states = []
+
+        for field in self.__get_field('250'):
+            footnote_text = self._handle_subfields_cardinality_max_one(self.__get_subfield_texts(field, 'a'))
```

Comment:
You could call this variable something else, just to make it clear.

Reply:
yes... that's copy-paste

Reply:
ok :-)